script/download_MNIST.py

```diff
@@ -6,29 +6,39 @@
 import subprocess
 import pickle
 
-## download train labels
+# download train labels
 print("Downloading train-labels-idx1-ubyte ......")
 http = urllib3.PoolManager()
 r = http.request('GET', 'http://yann.lecun.com/exdb/mnist/train-labels-idx1-ubyte.gz')
 data = gzip.decompress(r.data)
+# number of images is 4-8
```

Comment:
the images exist at these indices, correct? We have more than 4-8 images I think 😂

Reply:
4-8 contains the int of the number of images (ie the number 60,000) not all 60,000 images. Not sure if there is a better way to word this